Zenika/social-video-generatorz
# 31 integrate botz videos

## 🤔 Why?

To make the video editable, they must be integrated to the webapp

## 💻 How?

I've updated the home page by adding two preview player for the two videos. There is a button that redirect to a page where we will be able to changes the parameters of the video and generate it

## ✅ How to validate this PR?

- run `pnpm dev`. You should see two card with videos inside, a title and a button for each
- Click on a button. You should see a page with a video corresponding to the video you clicked on
